syz-verifier/main.go

```diff
@@ -521,15 +517,15 @@ func (srv *RPCServer) cleanup(poolIdx, vmIdx int) {
 	progs := srv.pools[poolIdx].vmRunners[vmIdx]
 	delete(srv.pools[poolIdx].vmRunners, vmIdx)
 	for _, prog := range progs {
+		if prog.done {
+			continue
+		}
+
+		prog.res = append(prog.res, &Result{Pool: poolIdx, Crashed: true})
```

Comment:
Ok, so now I think we can delete left, right? (no pun intended)
Also it seems we can now use newResult here.
And also move "prog.done = true" and "delete(srv.progs, prog.idx)" into newResult to avoid duplicating it in 2 places.

Reply:
yes, we can :)